web/src/helpers/useRainbow.js

```diff
@@ -1,30 +1,33 @@
 import React, {createContext, useContext, useState} from 'react'
+import {toPastel} from './colours'
 
-const gayStripes = ['#FF0000', '#FF5500', '#FDFF00', '#008400', '#2200FF', '#550087', '#D000FF']
-const transStripes = ['#55CDFC', '#F7A8B8', '#DDD', '#F7A8B8', '#55CDFC']
-const biStripes = ['#D9006F', '#D9006F', '#744D98', '#0033AB', '#0033AB']
-const aceStripes = ['#000000', '#A3A3A3', '#DDD', '#810082']
-const panStripes = ['#FF008E', '#FFD800', '#00B3FF']
+// stolen from flag
+const gayStripes = ['#FF5D7D', '#FF764E', '#FFC144', '#88DF8E', '#00CCF2', '#B278D3']
+const transStripes = ['#55CDFC', '#F7A8B8', '#DDD', '#F7A8B8', '#55CDFC'].map(toPastel)
+const biStripes = ['#D9006F', '#D9006F', '#744D98', '#0033AB', '#0033AB'].map(toPastel)
+const aceStripes = ['#000000', '#A3A3A3', '#DDD', '#810082'].map(toPastel)
+const panStripes = ['#FF008E', '#FFD800', '#00B3FF'].map(toPastel)
+// pastel enough
 const lesbianStripes = ['#B60063', '#C84896', '#E253AB', '#DDD', '#F0A7D2', '#D73F4F', '#990200']
 
 const allStripes = [
   gayStripes,
-  ...[transStripes, panStripes, biStripes, aceStripes, lesbianStripes].sort(
+  ...[transStripes, panStripes, biStripes, aceStripes, lesbianStripes]/* .sort(
```

Comment:
I am gonna add this back, for RANDOMNESS YAY

Reply:
oh, oops, that was for testing